lib/plugins/aws/package/compile/events/apiGateway/lib/hack/updateStage.test.js

```diff
@@ -395,6 +395,14 @@ describe('#updateStage()', () => {
     });
   });
 
+  function expectPatchOperation(patchOperation, callOffset) {
+    if (callOffset === undefined) {
```

Comment:
` if (!callOffset)` or even better we can set default in signature as `callOfset = 2`

Still, I see that we never pass this argument. What's the reason of introducing it? (before there was `2` hardcoded, and I think it's fine to keep it like that)

Reply:
It's true, since I never ended up overriding it, I just hardcoded it back to 2. Since it's code for specific tests this is fine and the arg can be reintroduced if it needs to be in the future.